Philips.CodeAnalysis.Benchmark/Program.cs

```diff
@@ -4,11 +4,6 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using BenchmarkDotNet.Attributes;
-using BenchmarkDotNet.Running;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
```

Comment:
Remove this file from the PR. The change serves no purpose, yet causes the build to fail.